front/gatsby/README.md

```diff
@@ -14,8 +14,8 @@ Install the dependencies using `npm`:
 
 ## Develop
 
-    $ npm run develop
+    $ npm start
 
-## Deploy
+## Build
 
-    $ npm run serve
+    $ npm run build

```

Comment:
```suggestion
    $ npm run build
    $ npm exec serve ./build
    
    # On older npm versions:
    $ npx serve ./build
```

Server will be available on [`localhost:5000`](http://localhost:5000).

Reply:
J'ai inclus ta suggestion :+1: